rancher_pr_deploy.sh

```diff
@@ -13,14 +13,31 @@ set -o pipefail
 
 declare -r NHSUK_GITHUB_URL="https://api.github.com/repos/nhsuk"
 
+find_latest_rancher_template() {
+
+  TEMPLATE_URL_BASE=$1
+  TEMPLATE_VERSION=0
```

Comment:
Because of `set -u` variables now need to be declared (using readonly flag as appropriate). See [this](https://github.com/nhsuk/ci-deployment/pull/4/commits/e510cf8c3f89f9df51f66850862965cb8074dc93) commit for rational for adding the setting. 
```
declare -r TEMPLATE_URL_BASE=$1
declare TEMPLATE_VERSION; TEMPLATE_VERSION=0
```
The script currently fails with '_./rancher_pr_deploy.sh: line 18: TEMPLATE_URL_BASE: readonly variable_' error